docs/cli.md

```diff
@@ -218,6 +219,19 @@ Usage:
 $ s2i rebuild <image name> [<new-tag-name>]
 ```
 
+# s2i generate
+The `s2i generate` command produces a Dockerfile using an existing S2I base 
+image that can be used to produce an image by any application supporting the
+format.
+
+The *image name* argument is in the URL-like, for example 
+`docker://docker.io/centos/nodejs-10-centos7`.
+
+Usage:
+
+```
+$ s2i generate <image name> <output file>
```

Comment:
Could you keep it consistent with the `s2i build..` docs and call it `<builder image>` ?

Reply:
Done in https://github.com/openshift/source-to-image/pull/1023/commits/38d86ba41a23fc0cdd094fcb12e5c08bd602ae52